Description: do not use -w flag when calling ninja, which requires ninja >= 1.6.0
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/tools/gn/bootstrap/bootstrap.py
+++ b/tools/gn/bootstrap/bootstrap.py
@@ -81,10 +81,10 @@ def main(argv):
       os.path.join(BOOTSTRAP_DIR, 'last_commit_position.h'), gn_build_dir)
   if options.jobs:
     subprocess.check_call(
-        ['ninja', '-C', gn_build_dir, 'gn', '-w', 'dupbuild=err', '-j'+str(options.jobs)])
+        ['ninja', '-C', gn_build_dir, 'gn', '-j'+str(options.jobs)])
   else:
     subprocess.check_call(
-        ['ninja', '-C', gn_build_dir, 'gn', '-w', 'dupbuild=err'])
+        ['ninja', '-C', gn_build_dir, 'gn'])
   shutil.copy2(os.path.join(gn_build_dir, 'gn'), gn_path)
 
   if not options.skip_generate_buildfiles:
